docs/examples/basic_usage/example_fisher_monte_carlo.py

```diff
@@ -78,7 +78,7 @@
 # Fisher and compute their matrix representations by multiplying them onto the
 # identity matrix:
 
-GGN = GGNLinearOperator(model, loss_function, params, data)
+GGN = GGNLinearOperator(model, loss_function, params, data).to_scipy()
```

Comment:
What is the reason to call `.scipy()` for `GGN` but not for `F`?

Reply:
This PR only changes the GGN linear operator to pure PyTorch. The Fisher linear operator is still a normal SciPy linear operator, therefore it does not need to be converted explicitly.

Reply:
ahh of course, thanks!